src/olympia/addons/widgets.py

```diff
@@ -45,8 +45,9 @@ def render(self, name, value, attrs=None):
         miscs = Category.objects.filter(misc=True).values_list('id', flat=True)
         for c in self.choices:
             if c[0] in miscs:
-                other = (c[0],
-                         _("My add-on doesn't fit into any of the categories"))
+                msg = ugettext(
+                    "My add-on doesn't fit into any of the categories")
```

Comment:
They're all over the place. To be honest, I started but eventually gave up changing them too all the time.